.github/workflows/action-deploy-apps.yml

```
@@ -0,0 +1,175 @@
+﻿name: Deploy apps
+
+env:
+  AZ_CLI_VERSION: 2.56.0
+
+on:
+  workflow_call:
+    secrets:
+      AZURE_CLIENT_ID:
+        required: true
+      AZURE_TENANT_ID:
+        required: true
+      AZURE_SUBSCRIPTION_ID:
+        required: true
+      AZURE_RESOURCE_GROUP_NAME:
+        required: true
+      AZURE_ENVIRONMENT_KEY_VAULT_NAME:
+        required: true
+      AZURE_CONTAINER_APP_ENVIRONMENT_ID:
+        required: true
+      AZURE_APP_INSIGHTS_CONNECTION_STRING:
+        required: true
+      AZURE_APP_CONFIGURATION_NAME:
+        required: true
+      AZURE_ADO_CONNECTION_STRING_SECRET_URI:
+        required: true
+
+    inputs:
+      region:
+        required: true
+        type: string
+      environment:
+        required: true
+        type: string
+      dryRun:
+        description: "If true, the deployment will be validated but not executed."
+        required: false
+        type: boolean
+        default: false
+      gitShortSha:
+        description: "Short SHA of the commit"
+        required: true
+        type: string
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.environment }}
```

```suggestion
  group: ${{ github.workflow }}-${{ inputs.environment }}-${{ inputs.dryRun }}
```

Kan de grupperes sånn? 🙈